hm_1.py

```diff
@@ -0,0 +1,25 @@
+"""
+Calculate sum of all digits of a number set in string.
+
+Example:
+
+num = �12345�
+
+Your script should calculate it like: 1+2+3+4+5 = 15
+
+num = �611091234512�
+result = 6+1+1+0+9+1+2+3+4+6+1+2 = 36
+"""
+
+
+def sum_str(n):
+    sum_line = sum(int(i) for i in str(n))
+    return ("Suma line {n} = {sum_line}".format(n=n, sum_line=sum_line))
```

Comment:
It's a bad idea.

I suggest returning just a final sum and format string when you're printing the result.

Reply:
Corrected

Reply:
As I can see, you're returning a formatted string instead of sum.